src/components/LHNOptionsList/LHNOptionsList.js

```diff
@@ -62,13 +132,38 @@ function LHNOptionsList({style, contentContainerStyles, data, onSelectRow, optio
      *
      * @return {Component}
      */
-    const renderItem = ({item}) => (
-        <OptionRowLHNDataWithFocus
-            reportID={item}
-            viewMode={optionMode}
-            shouldDisableFocusOptions={shouldDisableFocusOptions}
-            onSelectRow={onSelectRow}
-        />
+    const renderItem = useCallback(
+        ({item: reportID}) => {
+            const itemFullReport = reports[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`] || {};
+            const itemReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`];
+            const itemParentReportActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${itemFullReport.parentReportID}`];
+            const itemPolicy = policy[`${ONYXKEYS.COLLECTION.POLICY}${itemFullReport.policyID}`];
+            const itemTransaction = `${ONYXKEYS.COLLECTION.TRANSACTION}${lodashGet(
+                itemParentReportActions,
+                [itemFullReport.parentReportActionID, 'originalMessage', 'IOUTransactionID'],
+                '',
+            )}`;
+            const itemComment = draftComments[`${ONYXKEYS.COLLECTION.REPORT_DRAFT_COMMENT}${reportID}`] || '';
+            const participantPersonalDetailList = _.values(OptionsListUtils.getPersonalDetailsForAccountIDs(itemFullReport.participantAccountIDs, personalDetails));
```

Comment:
This misses an edge case where a user sent a last message in a room and then they leave the room. Then the report participant accountIDs do not include this user accountID and the user's email instead of their username is displayed in LHN. Issue #35477